src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-details/rbd-details.component.ts

```diff
@@ -10,18 +9,13 @@ import { RbdFormModel } from '../rbd-form/rbd-form.model';
 })
 export class RbdDetailsComponent implements OnChanges {
   @Input()
-  selection: CdTableSelection;
-  selectedItem: RbdFormModel;
+  selection: RbdFormModel;
   @Input()
   images: any;
   @ViewChild('poolConfigurationSourceTpl', { static: true })
   poolConfigurationSourceTpl: TemplateRef<any>;
 
   constructor() {}
 
-  ngOnChanges() {
-    if (this.selection.hasSelection) {
-      this.selectedItem = this.selection.first();
-    }
-  }
+  ngOnChanges() {}
```

Comment:
```suggestion
```
This can be removed. Tested it and there was no difference after removing it.

Reply:
I did run npm audit fix.

By removing ngOnChanges I'm getting an error that the type is missing, because of the implements statement. When I remove the implements and the import I'm getting the following error: 

```
Uncaught TypeError: directive.ngOnChanges is not a function
```

Reply:
When does this message show?

Reply:
When you try to open the rbd list

Reply:
I was able to reproduce it, but it only happened while I was editing the file.
After I forced a new build or did a prod build, it never showed again.

Reply:
huh, interesting.... Well then I can remove it :)